superset-frontend/src/views/CRUD/dashboard/DashboardCard.tsx

```diff
@@ -67,13 +67,11 @@ function DashboardCard({
 
   const menu = (
     <Menu>
-      {canEdit && openDashboardEditModal && (
+      {canEdit && handleDashboardEdit && (
         <Menu.Item
           role="button"
           tabIndex={0}
-          onClick={() =>
-            openDashboardEditModal && openDashboardEditModal(dashboard)
-          }
+          onClick={() => handleDashboardEdit && handleDashboardEdit(dashboard)}
```

Comment:
Can we use a regular `<a href="..." target="_blank">` instead of `onClick` event?

Reply:
I don't really like superset's gratuitous use of `target="_blank"`, I usually have too many tabs open and don't need any more.

Reply:
switched to using `<a />` but left out `target="_blank"` has users can achieve the same result with `CMD + click` or OS equivalent

Reply:
Opening in new tab is often helpful in context switching actions so users can return to previous page without losing context. It has been proven to be helpful for conversion time and time again on e-commerce websites.

While Superset is a different product, there are places where opening a new window makes sense, too. We should ask ourselves:

1.  Are users likely to go back to the previous page after finishing actions on the page they will open?
2. Does the previous page load slowly?
3. Is retaining scrolling position  on previous page important?

If the answer to any of these questions is "Yes", then we should open the link in a new page.

Reply:
With that said, I don't think opening in a new tab matters that much in this case. But just FYI, Tableau opens it in a new tab.

Reply:
We can add it, whatever we decide we should be consistent with https://github.com/apache/incubator-superset/pull/11667